packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -214,9 +216,43 @@ class TextMapPropagator {
     return this._config.tracePropagationStyle[mode].includes(name)
   }
 
+  _ensureTraceContextTakePrecedence (traceContext, spanContext, carrier) {
```

Comment:
nit: We can make these parameters a little more clear

```suggestion
  _ensureTraceContextTakePrecedence (w3cSpanContext, firstSpanContext, carrier) {
```